### ISSUE TITLE
Add Noteheads

### ISSUE THREAD

**Issue by drumnkyle** [drumnkyle/music-notation-core#109]
Add a `noteHead` property to `Note`. https://musescore.org/en/handbook/noteheads-0

**Comment by massimobio** [comment]
Should the ```noteHead``` property be on ```SpelledPitch``` rather then on ```Note```?
In percussion notation we might have two pitches with different note heads in one note.

**Comment by drumnkyle** [comment]
Yeah. That seems to make sense to me. Thanks for pointing it out.
